electron/config.ts

```diff
@@ -223,7 +224,7 @@ abstract class GlobalConfig {
       })
     })
 
-    const lutrisPath = `${homedir()}/.local/share/lutris`
+    const lutrisPath = `${home}/.local/share/lutris`
```

Comment:
I needed to change this to `homedir` because on flatpak it was looking inside `flatpakHome`, that's goes to the sandboxed home. But actually steam and lutris are installed outside this home, on the standard home.
So perhaps we can use the `homedir` everywhere except if it relies on the sandboxed home of flatpak